pyproject.toml

```diff
@@ -50,12 +50,14 @@ pytest = ">=6.0.0"
 pytest-cov = ">=2.6.1"
 codecov = "*"
 
-# These are used in the build script: for compiling the library (cmake/pybind11)
-# and for generating type stubs (mypy).
-cmake = "^3.25.0"
+black = "^22.12.0"

```

Comment:
Question: do we want to keep `black` as dependency in poetry or should we leave it to pre-commit?

Reply:
Good point. I added it initially so I can tell my IDE to use it for formatting, but I can also just rely on the system-wide Python's `black` for that. I've removed it from the dependency list.